crates/networking/p2p/peer_handler.rs

```diff
@@ -29,12 +27,21 @@ use crate::{
     snap::encodable_to_proof,
 };
 use tracing::info;
-pub const PEER_REPLY_TIMOUT: Duration = Duration::from_secs(45);
+pub const PEER_REPLY_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
wonder if this will affect snap sync, considering it might be requesting larger things. Fine to keep this though